.github/workflows/e2e-all-tests-parallel.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Terraform Job
         uses: hashicorp/setup-terraform@v1
         with:
-          terraform_version: 1.0.11
+          terraform_version: 1.1.7
```

Comment:
I think we should test both min and max of Terraform versions.

Reply:
Not sure why we need to test min versions, this could take a long time with very little benefit. Customers are generally looking for support for latest versions and here we are certifying that the latest 1.1.7 works without issues.